server/src/app.js

```diff
@@ -26,8 +26,6 @@ if (process.env.NODE_ENV === 'development') app.use(morgan('dev'));
 
 app.use('/api/v1/', router);
 
-if (process.env.NODE_ENV === 'development') app.use(morgan('dev'));
```

Comment:
Why did you remove morgan

Reply:
cause it written twice, i just removed the repeated line